src/lib/rule-finder.js

```diff
@@ -57,21 +63,23 @@ function _getPluginRules(config) {
   const plugins = config.plugins;
   /* istanbul ignore else */
   if (plugins) {
-    plugins.forEach(plugin => {
-      const normalized = normalizePluginName(plugin);
-      const pluginConfig = require(normalized.module);
-      const rules = pluginConfig.rules === undefined ? {} : pluginConfig.rules;
+    (ESLint.configType === 'eslintrc' ? plugins : Object.keys(plugins)).forEach(plugin => {
+      const normalized = ESLint.configType === 'eslintrc' ? normalizePluginName(plugin) : null;
+      const pluginConfig = normalized ? require(normalized.module) : null;
+      const rules = pluginConfig 
+        ? pluginConfig.rules === undefined ? {} : pluginConfig.rules
+        : plugins[plugin].rules === undefined ? {} : plugins[plugin].rules;
 
       Object.keys(rules).forEach(ruleName =>
-        pluginRules.set(`${normalized.prefix}/${ruleName}`, rules[ruleName])
+        pluginRules.set(`${normalized ? normalized.prefix : plugin}/${ruleName}`, rules[ruleName])
```

Comment:
```suggestion
        pluginRules.set(`${normalized ? normalized.prefix : plugin}/${ruleName}`, rules?.[ruleName])
```
then we don't need to set `rules` to an empty object.

(we might need to add the babel optional chaining plugin, which is fine)

Reply:
Since it seems that Babel6 causes an error and the only way is to raise it to Babel7, the implementation does not use an optional chain.
```
/home/study/workspace/eslint-find-rules/node_modules/babel-plugin-transform-optional-chaining/lib/index.js:12
  var _options$loose = options.loose,
```

"babel-plugin-transform-optional-chaining": "^7.0.0-beta.3"